mkdocs.yml

```diff
@@ -19,6 +19,7 @@ theme:
     repo: fontawesome/brands/github
     admonition:
       example: material/human-male-board
+      annotation: material/arrow-right-circle
```

Comment:
This doesn't seem to work, right? It's not an arrow which is used but a plus.

Reply:
Right sorry, I just fixed it. See the doc here > https://squidfunk.github.io/mkdocs-material/reference/annotations/.